### PR TITLE
Disable page and crop orientation

### PR DESCRIPTION
**Pull request by milosacimovic**
Hi @felixdittrich92 ,

I've merged your changes from disable orient prototype and slightly modified the function extract_rcrops which works better according to some of my internal tests.

### CODE REVIEW

**Comment by felixdittrich92** [comment]
Hi @milosacimovic :wave: 
Thanks i will have a look asap :+1:

**Comment by felixdittrich92** [comment]
API test fail is fine will success after merge

**Comment by felixdittrich92** [comment]
@milosacimovic Could you fix the mypy issue please https://github.com/mindee/doctr/actions/runs/11067610457/job/30751370517?pr=1735

**Comment by felixdittrich92** [comment]
Overwrites: #1723

File: doctr/models/classification/predictor/pytorch.py [hunk 1]
```diff
@@ -43,12 +43,16 @@ def forward(
...
+            # predictor is disabled
+            return [[0] * len(inputs), [0] * len(inputs), [1.0] * len(inputs)]
+
         processed_batches = self.pre_processor(inputs)
         _params = next(self.model.parameters())
         self.model, processed_batches = set_device_and_dtype(
             self.model, processed_batches, _params.device, _params.dtype
         )
-        predicted_batches = [self.model(batch) for batch in processed_batches]
+        predicted_batches = [self.model(batch) for batch in processed_batches if self.model is not None]
```
[the remark this answers is not in the document]
Comment (felixdittrich92) [reply doctr/models/classification/predictor/pytorch.py:55]: @milosacimovic Last change then we are good :+1:

[the remark this answers is not in the document]
Comment (milosacimovic) [reply doctr/models/classification/predictor/pytorch.py:55]: @felixdittrich92 that actually made more sense to me as well, but thought that being more explicit would help. anyways change is coming up.

**Review by felixdittrich92 — APPROVED** [review]
Thanks @milosacimovic

LGTM now :+1: 
Really nice and useful feature 🥳

**Comment by felixdittrich92** [comment]
CC @nikokks 
Using now with the `ocr_predictor` and `assume_straight_pages=False` and/or `straighten_pages=True` and/or `detect_orientation`

```
disable_crop_orientation=True,  # if all the text on doc is only horizontal
disable_page_orientation=True, # if your doc contains only small rotations (~ -45 -45)
```
Should also speed up processing a lot :)

**Pull request merged**